src/types/index.ts

```diff
@@ -14,6 +14,7 @@ export const enum Module {
 }
 
 export enum Section {
+    LandingPage = 0,
```

Comment:
did you reserve 0 on purpose, knowing you were eventually going to add this?

Reply:
yeah, it was also easier for having the page number system be 1 indexed.